provision-vm-service/provision-vm.asl

```diff
@@ -58,12 +58,6 @@
       "Default": "FailState"
     },
 
-    "RetryState": {
```

Comment:
I moved this down to match the order from the Choice state.  It didn't change the rendered graph like it had in the past.